main.go

```diff
@@ -247,6 +249,11 @@ func main() {
 		setupLog.Error(err, "failed to add/run bootstrap dependencies to the controller manager")
 		os.Exit(1)
 	}
+	clientset, clientErr := kubernetes.NewForConfig(mgr.GetConfig())
+	if err != nil {
+		setupLog.Error(clientErr, "failed to create kubernetes clientset")
+	}
+	reviewer := rbac.NewReviewer(clientset)
```

Comment:
we can now re-use this reviewer anywhere we want to see if a serviceaccount has a permission! cc @iblancasa

Reply:
Sounds good! I can use this for my PRs related to creating the RBAC stuff for the processors